packages/charts/src/BarChart/index.js

```diff
@@ -116,7 +116,9 @@ function BarChart({
 
   const handleMaxDimmesion = dimmension => {
     if (dimmension > maxLabelDimmension) {
-      setMaxLabelDimmesion(dimmension);
+      setTimeout(() => {
+        setMaxLabelDimmesion(dimmension);
+      }, 500);
```

Comment:
![Peek 2020-02-26 09-56](https://user-images.githubusercontent.com/1779590/75319763-51d10d80-587e-11ea-92c3-2a2c514a89af.gif)

Reply:
Are you changing height?
This is a separate issue.

I think labels in just columns are not setting proper width. Will be resolved later.